test/lit/passes/inlining-max-comibned-size.wat

```diff
@@ -0,0 +1,735 @@
+;; NOTE: Assertions have been generated by update_lit_checks.py --all-items and should not be edited.
+
+;; RUN: foreach %s %t wasm-opt -O3 -S -o - | filecheck %s --check-prefix=default
+;; RUN: foreach %s %t wasm-opt -O3 --inline-max-combined-binary-size=96 -S -o - | filecheck %s --check-prefix=imcbs96
+;; RUN: foreach %s %t wasm-opt -O3 --inline-max-combined-binary-size=90 -S -o - | filecheck %s --check-prefix=imcbs90
+;; RUN: foreach %s %t wasm-opt -O3 --inline-max-combined-binary-size=0 -S -o - | filecheck %s --check-prefix=imcbs0
```

Comment:
Check-prefixes should be in all-caps, just as a convention, so `IMCBS0` etc.

Reply:
ok. fixed.